### PR TITLE
pen-shortcode

### PR DESCRIPTION
**Pull request by n-orlowski**
adding shortcodes for components

### CODE REVIEW

File: content/en/design/core/_index.md [hunk 1]
```diff
@@ -73,152 +55,64 @@ These are the status indication colors of the system. When necessary, use white
...
+The default app font is <a href="https://www.google.com/get/noto/">Noto Sans</a>. It is free, open source, supports 800 languages and was specifically designed for good web legibility. It is bundled with the app so that all users see the same font regardless of their particular device, language, browser, etc. This ensures a consistent experience for all users.
 
-**Line length:** The optimal line length for body text is considered to be 50-60 characters per line on desktop and 30-40 on mobile, including spaces.
-
-**Alignment:** Text in the app should be left-aligned by default unless otherwise specified. 
-
-**Color:** Most text in the app should be the ```@text-normal-color: @gray-ultra-dark color```.
+Most text in the app should be the ```@text-normal-color: @gray-ultra-dark color```.
 The lighter text color ```@text-secondary-color: @gray-dark)``` is used for labels and condition card filters.
 Hyperlinked text color is ```@text-hyperlink-color: @blue-dark)```.
```
Comment (abbyad) [inline content/en/design/core/_index.md:65]: Are these meant to be inline code, surrounded by a single backtick?

File: content/en/design/core/_index.md [hunk 2]
```diff
@@ -73,152 +55,64 @@ These are the status indication colors of the system. When necessary, use white
...
 
-```
-H5 (small body)
-font-size: 14px (.875rem)
-line-height: 1.25rem (20px)
-```
-<br> 
-
-![H5](img/H5.png)
+        {{% pen id="PoZObmY" %}}
```
Comment (abbyad) [inline content/en/design/core/_index.md:77]: If the shortcode to use the id as the first param, then you could update all the shortcodes accordingly:
```suggestion
        {{% pen PoZObmY %}}
```

File: layouts/shortcodes/pen.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+
+
```
[the remark this answers is not in the document]
Comment (n-orlowski) [reply layouts/shortcodes/pen.html:3]: `pen` is probably short for codepen 😄 I copy and pasted and followed the shortcode documentation instructions very closely.. To rename would I rename the syntax and html file?

File: content/en/design/core/_index.md (hunk 1 above)
Comment (n-orlowski) [reply content/en/design/core/_index.md:65]: Yep..

**Comment by abbyad** [comment]
The codepen shortcode works again after my earlier edit broke it. A single unnamed ID argument can now be used.

Also, for some reason there were links that appeared broken on my local, so fixed those too. I'll let you review on your local to see if you want to tweak the content more before merging.

**Comment by n-orlowski** [comment]
Eek sorry accidentally requested another review while trying to figure out how to address the last change request

**Pull request merged**